metabase/metabase
# Add warning message when editing database permissions

### Description

Adds a warning message when the user tries to refresh/close/navigate away from `Metabase Admin > Edit Data Permissions`.

### How to verify

1. Go to Metabase Admin, and try to edit any database permissions (whether it is data or collections)
2. Try to refresh the page
3. You should see a browser popup asking if you want to leave the page.

### Checklist



- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/30415)
